src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -771,7 +767,7 @@ g.test('empty_bind_group_layouts_requires_empty_bind_groups,render_pass')
     for (let i = 0; i < bindGroupLayoutEntryCount; i++) {
       renderPass.setBindGroup(i, emptyBindGroup);
     }
-    renderPass.draw(0);
+    t.doRender(renderPass, 'draw', true);
```

Comment:
Should be parameterized over all 4 values in `kRenderCmds`

Reply:
Done.